ner_v2/tests/numeral/number_range/number_range_ner_tests.yaml

```diff
@@ -0,0 +1,661 @@
+tests:
+  en:
+  - _id_: en_1
+    message: I want more than 200 banana
+    outputs:
```

Comment:
Outputs don't have ids like they do in number detector tests yaml ?

Reply:
They do.

Reply:
Oops missed in sorting. That being said, should we sort keys? In my opinion, sorting makes it harder to read and understand the output.